card/introduction.mdx

```diff
@@ -1,87 +1,119 @@
 ---
-title: 'Introduction'
-description: 'Text, title, and styling in standard markdown'
+title: 'Overview'
+description: 'The SwyftPay Cards API provides a powerful, secure, and easy-to-use set of endpoints for card management, including creation, retrieval, freezing, and termination of cards. This guide will cover essential operations to help you manage card transactions within your applications.'
```

Comment:
I feel like this is too long, it can be simple as ``Swyftpay Card API allows businesses to create card for their customer to pay anywhere global``

Then ``The SwyftPay Cards API provides a powerful, secure, and easy-to-use set of endpoints for card management, including creation, retrieval, freezing, and termination of cards. This guide will cover essential operations to help you manage card transactions within your applications.`` can now come after like where you put documentation